DXRando/DeusEx/Classes/TrashCanCommon.uc

```diff
@@ -0,0 +1,69 @@
+class TrashCanCommon extends DXRBase;
+
+static function DestroyTrashCan(Containers trashcan)
+{
+    local int i;
+	local Vector loc;
+	local TrashPaper trash;
+	local Rat vermin;
+    local Containers trashbag;
+    local float scale;
+
+	// trace down to see if we are sitting on the ground
+	loc = vect(0,0,0);
+	loc.Z -= trashcan.CollisionHeight + 8.0;
+	loc += trashcan.Location;
+
+	// only generate trash if we are on the ground
```

Comment:
what's the reason for only spawning trash while on the ground?

Reply:
or I guess that comes from the vanilla code for spawning paper?

Reply:
That bit comes from vanilla. If trash paper floats to a wall, it either floats along it or crashed to the ground very quickly. Either way, it's much too obvious and silly if it's high up.

I tried changing it so trash paper floats down but it's not good enough.

Reply:
makes sense, the game obviously doesn't similar aerodynamic drag, so the paper would fall at the normal speed of gravity